docs/tutorials/intro_tutorial.rst

```diff
@@ -118,47 +119,83 @@ call it)
-a specific model run. Each model will contain multiple agents, all of
-which are instantiations of the agent class. Both the model and agent
-classes are child classes of Mesa’s generic ``Model`` and ``Agent``
-classes. This is seen in the code with ``class MoneyModel(mesa.Model)``
-or ``class MoneyAgent(mesa.Agent)``. If you want you can specifically
-the class being imported by looking at the
-`model <https://github.com/projectmesa/mesa/blob/main/mesa/model.py>`__
-or
-`agent <https://github.com/projectmesa/mesa/blob/main/mesa/agent.py>`__
-code in the mesa repo.
-
-Each agent has only one variable: how much wealth it currently has.
-(Each agent will also have a unique identifier (i.e., a name), stored in
-the ``unique_id`` variable. Giving each agent a unique id is a good
-practice when doing agent-based modeling.)
-
-There is only one model-level parameter: how many agents the model
-contains. When a new model is started, we want it to populate itself
-with the given number of agents.
-
-The beginning of both classes looks like this:
+This includes importing of dependencies needed for the tutorial.
 
 .. code:: ipython3
 
     import mesa
 
+    # Visualization tools needed to explore the model.
+    import matplotlib.pyplot as plt
+    import numpy as np
+    import pandas as pd
+    import seaborn as sns
+
+Create Agent
+~~~~~~~~~~~~
+
+First create the agent. As the tutorial progresses, more functionality
+will be added to the agent.
+
+Background: Agents are the individual entities that act in the model.IT
+is a good modeling practice to make certain each Agent can be uniquely
+identified.
+
+Model Specific Information: Agents are the individuals that exchange
+money, in this case the amount of money an individual agent has is
+represented as wealth. Additionally, agents each have a unique
+identifier.
+
+Technical Details: This is done by creating a new class (or object) that
+extends ``mesa.Agent`` creating a subclass of the ``Agent`` class from
+mesa. The new class is named ``MoneyAgent``. The technical details about
+the agent object can be found in the `mesa
+repo <https://github.com/projectmesa/mesa/blob/main/mesa/agent.py>`__.
+
+The ``MoneyAgent`` class is created with the following code:
+
+.. code:: ipython3
 
     class MoneyAgent(mesa.Agent):
         """An agent with fixed initial wealth."""
 
         def __init__(self, unique_id, model):
+            # Pass the parameters to the parent class.
             super().__init__(unique_id, model)
+
+            # Create the agent's variable and set the initial values.
             self.wealth = 1
 
+Create Model
+~~~~~~~~~~~~
+
+Next create the model. Again as the tutorial progresses, more
+functionality will be added to the model.
+
+Background: The model can be visualized as a grid containing all the
+agents. The model creates, holds and manages all the agents on the grid.
+Running the model will work through all the time cycles.
```

Comment:
I find this unclear in how it describes the time evolution of the model. It's about describing what a discrete-event simulation is, or rather, assume that the reader knows or can search the meaning of the term.